src/coreclr/jit/codegenarm64.cpp

```diff
@@ -9564,4 +9564,47 @@ void CodeGen::instGen_MemoryBarrier(BarrierKind barrierKind)
     }
 }
 
+//-----------------------------------------------------------------------------------
+// genCodeForMadd: Emit a madd/msub (Multiply-Add) instruction
+//
+// Arguments:
+//     tree - GT_MADD tree where op1 or op2 is GT_ADD
+//
+void CodeGen::genCodeForMadd(GenTreeOp* tree)
+{
+    assert(tree->OperIs(GT_MADD) && varTypeIsIntegral(tree) && !(tree->gtFlags & GTF_SET_FLAGS));
+    genConsumeOperands(tree);
+
+    GenTree *a, *b, *c;
```

Comment:
Nit: our style guide forbids such one-line declarations.
```suggestion
    GenTree* a;
    GenTree* b;
    GenTree* c;
```